qa.py

```diff
@@ -234,6 +234,9 @@ def transform_levels(entry):
                         )
 
                         for index, r in enumerate(result["results"]):
+                            if r.get("failMsg"):
```

Comment:
As I understood from reading the JS code, in the JS version we don't do this? I saw this on line 279 of qa.ts:

```
          const failStr = r?.failMsg ? `failed! ${r.failMsg}` : '';
```

And then the output has failStr:

```
          console.log(
            `[${indexStr}] ${rfqIdStr.padEnd(
              maxRfqIdLength,
              ' '
            )}  ${baseAmountStr}  ${quoteAmountStr}  ${expectedAmountStr}  ${deviationStr}  ${feesStr}  ${failStr}`
          );
```

I'm just wondering if I missed something here? If not, should we update the JS version to behave the same way?

Reply:
@mib-hashflow has more context on the JS version -- I was under the impression that we do show detailed errors in that one.

My symptom FWIW: one quote failing would result in the entire script failing. In reality, it's likely that MMs will fail SOME quotes and the fail reasons are likely to be useful to them in assessing the root cause.